### PR TITLE
Added a note

### PR DESCRIPTION
**Pull request by joinimran**
Added a note to keep room and equipment mailbox secure. 

Problem: https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/issues/770

Reference PR: https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/pull/794

### CODE REVIEW

**Comment by joinimran** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/manage/room-and-equipment-mailboxes.md [hunk 1]
```diff
@@ -93,6 +93,9 @@ To set up a room or equipment mailbox, go to the Microsoft 365 admin center. (Yo
   
 6. If you made changes, select **Save** and then **Close**.
 
+> [!Note]
+> To keep your room and equipment mailboxes secure, block sign-in to these mailboxes. See details [here](https://docs.microsoft.com/office365/admin/email/create-a-shared-mailbox?view=o365-worldwide#block-sign-in-for-the-shared-mailbox-account).
```
Comment (JohanFreelancer9) [inline Office365-Admin/manage/room-and-equipment-mailboxes.md]:
```suggestion
> To keep your room and equipment mailboxes secure, block sign-in to these mailboxes. For more information, see [Block sign-in for the shared mailbox account](https://docs.microsoft.com/office365/admin/email/create-a-shared-mailbox?view=o365-worldwide#block-sign-in-for-the-shared-mailbox-account).
```

**Comment by joinimran** [comment]
Thanks, @JohanFreelancer9 .

Ready to go live @mypil

**Comment by mypil** [comment]
@cmcatee-MSFT - PR has been copy edited and is ready for final review. This is based on issue #770 and related to your recommendation on PR #794. By the way, the author metadata for this doc still shows @twernermsft, will you be the new doc owner? Please advise so we can update the metadata.

Thank you.

cc: @kenwith

**Comment by cmcatee-MSFT** [comment]
@mypil Please do not change the author metadata.

**Pull request merged**